src/admin-views/aggregator/settings.php

```diff
@@ -119,7 +119,8 @@
 
 	$stop_running_processes_message = sprintf(
 		__( 'If you want to stop and clear current asynchronous import processes %1$s.', 'the-events-calendar' ),
-		sprintf( '<a href="' . add_query_arg( array( Tribe__Events__Aggregator__Processes__Queue_Control::CLEAR_PROCESSES => 1 ) ) . '">%s</a>',
+		sprintf( '<a href="%s">%s</a>',
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 486c586daefabf54d883a940e810b6c5</sub>

Reply:
we should address this

Reply:
fixed